topiary-queries/queries/rust.scm

```diff
@@ -205,3 +205,14 @@
 (
   "," @prepend_antispace
 )
+
+; append trailing comman to newline delimited arguments
+(arguments
+  (#delimiter! ",")
+  (_) @append_multiline_delimiter
+  .
+  ","? @do_nothing
+  .
+  ")"
```

Comment:
There's an idempotence check failure here that's likely clashing with the `@append_spaced_softline` after the comma:
https://github.com/tweag/topiary/blob/53eeec81ad882e5fb86c0aff16362736eb9bb74a/topiary-queries/queries/rust.scm#L108-L116

Logically, the precedence should be to handle the append delimiter before handling the sofline.